api/src/lib/helpers.ts

```diff
@@ -713,7 +713,7 @@ export const formatAffiliationName = (affiliation: I.MappedOrcidAffiliation): st
     }${organization.address.country}`;
 };
 
-const doiLinkBase = `https://doi.org/`;
+const doiLinkBase = `https://${process.env.STAGE === 'prod' ? 'doi.org' : 'handle.test.datacite.org'}/`;
```

Comment:
Possible alternative, put the doi url in environment variables. can just use ${process.env.DOI_URL} for example. Other than code style preferences, benefit is incase that url changes or you need to introduce a different url for int in the future, it's easy to change in env vars.

Reply:
That was my initial plan actually; I thought it would be easier not to have it in the env vars just to keep that list as short as it can be and give new people 1 less thing to look up / understand before they can get started. It's true about the ease of changing though.

Reply:
Ah interesting thought, maybe I should learn a lesson from this, I just make the env files bigger and bigger then regret it when I have to do a clean clone of the repo.